drivers/SmartThings/matter-sensor/src/init.lua

```diff
@@ -23,6 +23,41 @@ local function device_init(driver, device)
   device:subscribe()
 end
 
+local function do_configure(driver, device)
+  print("do_configure")
```

Comment:
nit:print statement

Reply:
Removed in 9473139b5f9e4b8c2b2466c509e3cd90ef9b251e